### DISCUSSION TITLE
How can I use Function key in custom key mappings file?

### DISCUSSION THREAD

**Discussion by thesanjithkumar** [NvChad/NvChad#2634] [Q&A]
I wanted to know if I could use functions keys for mapping some dap debugging commands like F9 for toggling breakpoints. etc

**Comment by siduck** [comment]
how would you do that in neovim generally?

**Reply by thesanjithkumar** [reply]
Ya I guess, since I'm a new user, I don't have much experience with neovim and nvchad.
[https://youtu.be/mh_EJhH49Ms?si=WEy5AV1CKDs5QWKE](Setup Tutorial video which I foloewed),
I wanted to use function keys instead of <leadderdb>.

**Reply by siduck** [reply]
ask at r/neovim then , please read all of neovim docs related to mappings as well as nvchad docs too

**Reply by thesanjithkumar** [reply]
Oh ok, thanks for the help
